### PR TITLE
i18n(pt-BR): update `styling.md` page

### PR DESCRIPTION
**Pull request by victor0x16**
PR #633 added new content to styling page.

### CODE REVIEW

**Comment by victor0x16** [comment]
Yep! Now is all green :)

`import styles from an npm package`  was/is very useful during hackathon.

File: src/pages/pt-BR/guides/styling.md [hunk 1]
```diff
@@ -111,7 +113,47 @@ import '../estilos/utils.css';
 
 `import` de CSS por meio de ESM é suportado dentro de qualquer arquivo JavaScript, incluindo componentes JSX como React e Preact. Isto pode ser útil para escrever estilos por componente de forma granular para seus componentes React.
 
-### Inclua uma folha de Estilos Externa
+### Importe uma Folha de Estilos de um Pacote do NPM
+
+Você talvez precise incluir uma folha de estilos de um pacote externo. Isso é especialmente comum para utilidades como [Open Props](https://open-props.style/). Se seu pacote **recomenda usar uma extensão de arquivo** (ex.: `nome-do-pacote/estilos.css` ao invés de `nome-do-pacote/estilos`), isso deve funcionar como qualquer importação de uma folha de estilos local.
```
Comment (Yan-Thomas) [inline src/pages/pt-BR/guides/styling.md:118]:
```suggestion
Você talvez precise incluir uma folha de estilos de um pacote externo. Isso é especialmente comum para utilitários como [Open Props](https://open-props.style/). Se seu pacote **recomenda usar uma extensão de arquivo** (ex.: `nome-do-pacote/estilos.css` ao invés de `nome-do-pacote/estilos`), isso deve funcionar como qualquer importação de uma folha de estilos local.
```

File: src/pages/pt-BR/guides/styling.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Astro foi desenvolvido pensando em tornar a estilização e a escrita de CSS fá
 
 ## Estilização em Astro
 
-Estlizar um componente Astro é tão fácil quanto adicionar uma tag `<style>` no seu componente ou template de página. E quando você coloca uma tag `<style>` dentro de um componente astro, Astro vai detectar o CSS e manipular os estilos para você automaticamente.
+Estilizar um componente Astro é tão fácil quanto adicionar uma tag `<style>` no seu componente ou template de página. E quando você coloca uma tag `<style>` dentro de um componente astro, Astro vai detectar o CSS e manipular os estilos para você automaticamente.
```
Comment (Yan-Thomas) [inline src/pages/pt-BR/guides/styling.md:15]:
```suggestion
Estilizar um componente Astro é tão fácil quanto adicionar uma tag `<style>` no seu componente ou template de página. E quando você coloca uma tag `<style>` dentro de um componente Astro, Astro vai detectar o CSS e manipular os estilos para você automaticamente.
```

File: src/pages/pt-BR/guides/styling.md [hunk 3]
```diff
@@ -111,7 +113,47 @@ import '../estilos/utils.css';
...
+    ssr: {
+      noExternal: ['nome-do-pacote'],
+    }
+  }
+})
+```
+
+> Nota: Isso é uma configuração [específica do Vite](https://vitejs.dev/config/#ssr-noexternal) que não tem relação com (ou necessita de) [SSR do Astro](/pt-BR/guides/server-side-rendering/).
+
+Agora, você está livre para importar `nome-do-pacote/normalize`. Isto será bundled e otimizado pelo Astro como qualquer outra folha de estilos local.
```
Comment (Yan-Thomas) [inline src/pages/pt-BR/guides/styling.md:146]:
```suggestion
Agora, você está livre para importar `nome-do-pacote/normalize`. Isto passará por bundle e será otimizado pelo Astro como qualquer outra folha de estilos local.
```

**Review by Yan-Thomas — CHANGES_REQUESTED** [review]
Thanks for your work! I added some suggestions.

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**